addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java

```diff
@@ -186,4 +207,35 @@ public String handleCallBack(HttpMessage msg) throws ApiException {
         // Will be accessed via a GET as part of the browser ext initiation
         return "";
     }
+
+    private void validateExportPath(String exportPath) throws ApiException {
+        try {
+            Path path = Paths.get(exportPath).toAbsolutePath().normalize();
+
+            // Check directory exists and is writable
```

Comment:
Remove these comments, if you think the code needs clarification extract a method.